packages/amplify-cli-core/src/state-manager/pathManager.ts

```diff
@@ -199,7 +199,11 @@ export class PathManager {
 
   private createNotInitializedError = (): Error => {
```

Comment:
It might be worth making this function reusable since this pattern is repeated a few times.

Reply:
Noted, and pushed!